vidarr/wdl.py

```diff
@@ -255,39 +254,45 @@ def read_output(output: WDL.Decl):
             if isinstance(output_metadata, dict) and 'vidarr_label' in output_metadata:
                 vidarr_label = output_metadata['vidarr_label']
 
-                # Define the pattern for searching and replacing the output block
-                pattern = r"(?:workflow)([\s\S]*?)(?:output\s*{)([\s\S]*?)(?:\}\s*\n)"
+                if isinstance(output.type, WDL.Type.File(optional=False)):
```

Comment:
ran this on test/empty.wdl and got `TypeError: isinstance() arg 2 must be a type, a tuple of types, or a union` on this line